backend/packages/Upgrade/src/api/services/StratificationService.ts

```diff
@@ -56,10 +58,13 @@ export class StratificationService {
           factor: factors.stratificationFactorName,
           value: 'N/A',
           count: 0,
+          experimetIds: factors.experiment[0].id,
```

Comment:
Why returning only single experimentId here?

Reply:
This is part of unused code now, I will remove the remainingFactors variable, its not used now.